run_pipeline_lightgcn.py

```diff
@@ -180,6 +179,7 @@ def train(config: LightGCNConfig):
             scheduler.step()
 
     if config.show_graph:
+        import matplotlib.pyplot as plt
         iters = [iter * config.eval_every for iter in range(len(train_losses))]
         plt.plot(iters, train_losses, label="train")
         plt.plot(iters, val_losses, label="validation")
```

Comment:
Do we need this at all if you use wandb?

Reply:
lightgcn is not wired up with wandb, as far as I know!